src/components/generators/schema/schema-service.ts

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
why don't register in jsonforms-schemagenerator just like you do it for renderer and thus remove the schema/uischema folders which would then contain exactly one class?